examples/plotting/grid_plotting.py

```diff
@@ -0,0 +1,49 @@
+"""
+==========================
+Plotting a coordinate grid
+==========================
+
+This example demonstrates how you can draw a grid of solar coordinates on top
+of a map.
+"""
+import matplotlib.pyplot as plt
+
+import astropy.units as u
+
+import sunpy.data.sample
+import sunpy.map
+
+################################################################################
+# Let's start with a sample AIA image.
+aia_map = sunpy.map.Map(sunpy.data.sample.AIA_171_IMAGE)
+
+################################################################################
+# Now we can plot the image, and add grids in heliographic Stonyhurst and
+# heliographic Carrington coordinates.
+fig = plt.figure()
+ax1 = fig.add_subplot(121, projection=aia_map)
+aia_map.plot(axes=ax1)
```

Comment:
Can you use `clip_interval` or `vmin`/`vmax`?  The AIA image is too dark for my liking

Reply:
I used `clip_interval` for both.